WalletWasabi/WabiSabi/Client/CoinJoinClient.cs

```diff
@@ -1002,16 +1002,73 @@ private async Task<IEnumerable<TxOut>> ProceedWithOutputRegistrationPhaseAsync(u
 		// Calculate outputs values
 		var constructionState = roundState.Assert<ConstructionState>();
 
+		var paymentsToBatch = (await DestinationProvider.GetPendingPayments(roundParameters,registeredAliceClients).ConfigureAwait(false)).ToList();
 		AmountDecomposer amountDecomposer = new(roundParameters.MiningFeeRate, roundParameters.AllowedOutputAmounts, Constants.P2wpkhOutputVirtualSize, Constants.P2wpkhInputVirtualSize, (int)availableVsize);
```

Comment:
the amount decomposee probably needs to be refactored to be able to handle different script types but this is outof scope for this PR

Reply:
Yes, that's something that I have to do after the support for taproot is merged. Here is what comes next: https://github.com/zkSNACKs/WalletWasabi/discussions/9216